src/Persistence.Tests/MsApp/MsappArchiveTests.cs

```diff
@@ -4,12 +4,25 @@
 using System.IO.Compression;
 using Microsoft.PowerPlatform.PowerApps.Persistence;
 using Microsoft.PowerPlatform.PowerApps.Persistence.MsApp;
+using Microsoft.PowerPlatform.PowerApps.Persistence.Yaml;
+using Moq;
```

Comment:
It is not recommended to use Moq because of serious privacy concerns: https://github.com/devlooped/moq/issues/1372